sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -523,6 +523,8 @@ case class JsonToStructs(
   // can generate incorrect files if values are missing in columns declared as non-nullable.
   val nullableSchema = if (forceNullableSchema) schema.asNullable else schema
 
+  val unpackArray: Boolean = options.get("unpackArray").map(_.toBoolean).getOrElse(false)
```

Comment:
private? (This is not related to this pr though, `nullableSchema` also can be private?)

Reply:
Can you make the option `unpackArray` case-insensitive?

Reply:
If we add this new option here, I feel we'd be better to document somewhere (e.g., `sq/functions.scala`)